<launch>
    <!-- Only node from the package -->
    <node pkg="ping_driver" name="ping_driver" type="pingDriver.py">

        <!-- Controls whether node reads from ping itself or from a fake data stream -->
        <param name="shouldEmulateData" value="True" />

        <!-- Ping Startup Parameters -->
        <param name="ping_enabled" value="False" />
        <param name="ping_frequency" value="10" />
        <param name="speed_of_sound" value="1498" />
        <param name="auto" value="True" />
        <param name="scan_start" value="0" />
        <param name="scan_length" value="10" />
        <param name="gain" value="0" />

    </node>

</launch>
